src/mpid/ch4/generic/mpidig_send.h

```diff
@@ -171,6 +233,34 @@ MPL_STATIC_INLINE_PREFIX int MPIDIG_mpi_send(const void *buf,
     return mpi_errno;
 }
 
+#undef FUNCNAME
+#define FUNCNAME MPIDIG_mpi_send_coll
+#undef FCNAME
+#define FCNAME MPL_QUOTE(FUNCNAME)
+MPL_STATIC_INLINE_PREFIX int MPIDIG_mpi_send_coll(const void *buf,
```

Comment:
This API should not have `_mpi_` in the name. The intention is for ch4 APIs with `_mpi_` to map directly to MPI functions.

Reply:
Yep. Good point. I'll take this out.

I'll also drop the `FUNCNAME` definition.